ed25519/src/lib.rs

```diff
@@ -402,6 +404,49 @@ impl<'de> Deserialize<'de> for Signature {
     }
 }
 
+#[cfg(all(feature = "serde", feature = "serde_bytes"))]
```

Comment:
Minor nit, but these are redundant as `serde_bytes` implies `serde`

```suggestion
#[cfg(feature = "serde_bytes")]
```